### ISSUE TITLE
[cmake] coverage shows things under /usr/local/include (Trac #1820)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1812]
[cmake] coverage shows things under /usr/local/include. owner: nega

Migrated from https://code.icecube.wisc.edu/ticket/1820
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:12:38", 
    "description": "[cmake] coverage shows things under /usr/local/include. owner: nega", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067158057333", 
    "component": "cmake", 
    "summary": "[cmake] coverage shows things under /usr/local/include", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2016-08-15T16:59:21", 
    "milestone": "", 
    "owner": "nega", 
    "type": "rumor / allegation"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160815T20:23:15: nega commented:

> no it doesn't, but it does show libs under `/cvmfs`

**Comment by icecube-trac** [comment]
Trac update at 20160815T20:24:24: david.schultz commented:

> It does when I run it locally using a boost from `/usr/local/include/boost`.  We really shouldn't care about anything not in I3_SRC.

**Comment by icecube-trac** [comment]
Trac update at 20160815T20:28:26: 
* nega commented:

> In changeset 2650:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="2650"
> add some more excludes to lcov's reporting
> 
> closes #1812
> ```


* nega changed status from "new" to "closed"
* nega changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190112T00:24:23: nwhitehorn commented:

> In changeset 170315:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="170315"
> add some more excludes to lcov's reporting
> 
> closes #1812
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:12:38: 
* olivas changed milestone from "August 2016" to ""
* olivas commented:

> Milestone August 2016 deleted
